pandas/tests/util/test_deprecate.py

```diff
@@ -0,0 +1,62 @@
+from textwrap import dedent
+
+import pytest
+
+from pandas.util._decorators import deprecate
```

Comment:
are these inplace of tests_deprecate_kwarg.py? (this is pretty new)

Reply:
No, they are different. We have an old (I guess) `deprecate` that is used to deprecate `argmax` (and `argmin`) with `deprecate('argmax', idmax)`.

The function was not tested, and I had to modify it, because it was injecting the `.. deprecate::` directive in a way that the docstring format was wrong.